### ISSUE TITLE
Would you be open to PR to provide a custom RegionLabel in the props?

### ISSUE THREAD

**Issue by ThibautGeriz** [waoai/react-image-annotate#28]
Hi,

I really like your library, thank you for open sourcing it. 
Our use case requires us to tag more than just a string and a list of string. Would you accept a PR when we give a custom form and use it instead of the provided `RegionLabel` ?

Cheers

**Comment by seveibar** [comment]
Hi @ThibautGeriz,

Do you mean a custom html form? What if we allowed JSON like this:

```javascript
[
{ "type": "radiogroup", name: "Is valid?", options: ["Valid", "Invalid"] },
{ "type": "text", name: "What type of object?" }
]
```

We may be able to list such a form on the sidebar if it gets too long. Let me know what you think.

**Comment by ThibautGeriz** [comment]
Our use case has logic between the inputs so I was thinking giving a react component in the props with a custom logic within. It will implement all the callback and props that you are giving.
By default the library would use `RegionLabel` unless the prop was provided.

I have already started to work a prototype and test it with our platform, your code is easy to read and the changes are small. I will open a PR soon as I would rather have that merged instead of us maintaining our custom fork.

**Comment by seveibar** [comment]
Thanks!

**Comment by seveibar** [comment]
merged in #32
